tree/dataframe/test/dataframe_display.cxx

```diff
@@ -635,3 +635,54 @@ TEST(RDFDisplayTests, PrintWideTables3)
    "+-----+--------------------------------------------------------------------------------------------------------+-----+\n");
 }
 
+TEST(RDFDisplayTests, DisplayZeroEntries)
+{
+   ROOT::RDataFrame d{1};
+   auto dd = d.Define("b1", [] { return 42; }).Display<int>({"b1"}, 0);
+
+   // Testing the std output printing
+   std::cout << std::flush;
+   // Redirect cout.
+   std::streambuf *oldCoutStreamBuf = std::cout.rdbuf();
+   std::ostringstream strCout;
+   std::cout.rdbuf(strCout.rdbuf());
+   dd->Print();
+   // Restore old cout.
+   std::cout.rdbuf(oldCoutStreamBuf);
+
+   EXPECT_EQ(strCout.str(), "+-----+----+\n"
+                            "| Row | b1 | \n"
+                            "+-----+----+\n");
+}
+
+TEST(RDFDisplayTests, DisplayLimitWithOtherOps)
+{
+   ROOT::RDataFrame d{10};
+   auto d1 = d.Define("b1", [] { return 42; });
```

Comment:
A small addition I would suggest to this test is to also Define an extra column, but not Display it.

So that we show that other Actions do not spoil the list of columns in Display.